modules/metalsmith/doc_assets.js

```diff
@@ -0,0 +1,168 @@
+'use strict';
+/* eslint-env es6 */
+
+
+const {info, error} = require('./../debugger')('nx_assets');
+const fs = require('fs');
+const crypto = require('crypto');
+const Nuxeo = require('nuxeo');
+const cheerio = require('cheerio');
+
+const map = {
+    // 'a'     : 'href',
+    // 'link'  : 'href',
+    // 'script': 'src',
+    'img'  : 'src',
+    'video': 'src',
+    'audio': 'src'
+    // 'source': 'src'
+};
+
+
+const checksum = function (str, algorithm, encoding) {
```

Comment:
Perhaps we could make use of ES2015 default parameters and arrow functions, something like:
```const checksum = (str, algorithm = 'md5', encoding = 'utf8') => crypto....```

Reply:
Same thing for other functions below...
`const is_html = (file) => /\.html?/.test(file);`